package-overrides/npm/engine.io-client@1.5.1.json

```diff
@@ -1,15 +0,0 @@
-{
-  "dependencies": {
-      "xmlhttprequest" : "git+https://github.com/rase-/node-XMLHttpRequest.git#add/ssl-support",
-      "has-cors": "1.0.3",
-      "ws": "0.7.1",
-      "component-emitter": "1.1.2",
-      "indexof": "0.0.1",
-      "engine.io-parser": "1.2.1",
-      "debug": "2.1.3",
-      "parseuri": "0.0.4",
-      "parsejson": "0.0.1",
-      "parseqs": "0.0.2",
-      "component-inherit": "0.0.3"
-  }
-}
```

Comment:
Rather than deleting, we should create a new override for the later version that this isn't needed on.